app/helpers/tasks_helper.rb

```diff
@@ -1,2 +1,63 @@
 module TasksHelper
+
```

Comment:
This could be refactored, but for now it's only complaining about duplication... we'll come back to it later